extensions/amp-story/validator-amp-story.protoascii

```diff
@@ -844,6 +844,29 @@ tags: {  # <amp-story-page-attachment> with no href
     value: "dark"
     value: "light"
   }
+  attrs: {
+    name: "cta-text"
+  }
+  attrs: {
```

Comment:
This will allow relative URLs for `cta-img` and `cta-img-2` below as that is the [default for validator](https://github.com/ampproject/amphtml/blob/master/validator/validator.proto#L62). Is that desired? Should AMP caches rewrite these URLs to be absolute?

Reply:
See cl/361937561. Are we confident that these attribute names aren't changing: `cta-img` and `cta-img-2`?

Reply:
Ok, I'll bite and bikeshed the name, since this is going to be proliferated across codebases.

Let's use `image` instead of `img`, to align with the [interactive component result API](https://github.com/ampproject/amphtml/tree/master/extensions/amp-story-interactive#multi-page-component-apis).  @honeybadgerdontcare do we need to support that in the transformer as well?

Reply:
Yes, they'll stay with those names. We considered different approaches but this is what we need

Reply:
changed to `cta-image` and `cta-image-2`

Reply:
Yes, we'll need to support that explicitly. I'll update the cl.

These are the attributes regardless of which tag they are on that get made absolute:
  * background
  * href
  * xlink:href
  * poster
  * src
  * srcset

Reply:
They'll be `data-cta-image` and `data-cta-image-2`, actually

Reply:
Updated transformers to reflect `data-cta-image` and `data-cta-image-2`.
Updated the protoascii here adding `allow_relative: false` until the transformers are in prod.

Reply:
final update here: we used `cta-image` and `cta-image-2`. Thank you so much David!